src/utils/GetTestResults.ts

```diff
@@ -1,19 +1,12 @@
 import * as dateFns from "date-fns";
 import * as _ from "lodash";
+import {TEST_VERSION} from "../assets/Enums";
+import {ITestResult} from "../models/ITestResult";
 
 
 export class GetTestResults {
   public static validateDates(fromDateTime: string | number | Date, toDateTime: string | number | Date) {
-    return _.isDate(new Date(fromDateTime)) && _.isDate( new Date(toDateTime)) && _.isFinite((new Date(fromDateTime)).getTime()) && _.isFinite((new Date(toDateTime)).getTime());
-  }
-
-  public static removeTestResultId(testResults: Array<{ testResultId: string | number; }>) {
-    if (testResults.length > 0) {
-      for (const [index, testResult] of testResults.entries()) {
-        delete testResults[index].testResultId;
-      }
-    }
-    return testResults;
+    return _.isDate(new Date(fromDateTime)) && _.isDate(new Date(toDateTime)) && _.isFinite((new Date(fromDateTime)).getTime()) && _.isFinite((new Date(toDateTime)).getTime());
```

Comment:
can reduce the import size as `isFinite` and `isDate` only used.

Reply:
done